### PR TITLE
RHINENG-7611: Add bootc_status

### PR DESCRIPTION
**Pull request by JoySnow**
[RHINENG-7611 ](https://issues.redhat.com/browse/RHINENG-7611)

### CODE REVIEW

**Review by kruai — APPROVED** [review]
LGTM!

**Comment by kruai** [comment]
@JoySnow Based on the linked Jira, I think this looks good. Is it OK if I merge it now?

**Review by FabriciaDinizRH — APPROVED** [review]

File: schemas/system_profile/v1.yaml [hunk 1]
```diff
@@ -191,6 +191,22 @@ $defs:
...
         type: boolean
         example: false
+  BootcStatusImageInfo:
+    description: Object containing image data from command bootc status
+    type: object
+    properties:
+      image:
+        description: Name of the image
+        type: string
+        example: "quay.io/centos-bootc/fedora-bootc-cloud:eln, 192.168.0.1:5000/foo/foo:latest"
```
Comment (beav) [inline schemas/system_profile/v1.yaml:201]: This is two examples separated by comma, not one. Sorry for the confusion

File: schemas/system_profile/v1.yaml [hunk 2]
```diff
@@ -191,6 +191,22 @@ $defs:
...
+        maxLength: 128
+        x-wildcard: true
+      image_digest:
+        description: Digest of the image
+        type: string
+        example: "sha256:806d77394f96e47cf99b1233561ce970c94521244a2d8f2affa12c3261961223, sha256:92e476435ced1c148350c660b09c744717defbd300a15d33deda5b50ad6b21a0"
```
Comment (beav) [inline schemas/system_profile/v1.yaml:207]: This is also two examples, same as above

**Comment by JoySnow** [comment]
Thanks for the note, @beav . 

And, thanks for the review, @kruai , @FabriciaDinizRH . 

I am okay to merge this PR.

**Pull request merged**